packages/module/src/DetailsPageHeader/DetailsPageHeader.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Split,
+  SplitItem,
+  TextContent,
+  Text,
+  TextVariants,
+  Label,
+} from '@patternfly/react-core';
+import React from 'react';
+import {
+  ActionButtonProps,
+  ActionButtons,
+  ActionMenu,
+  ActionMenuProps,
+  BreadcrumbProps,
+  Breadcrumbs,
+} from './utils';
+import '@patternfly/react-styles/css/utilities/Spacing/spacing.css';
+
+export interface PageHeading {
+  /** Title for page heading */
+  title: string;
+  /** Optional icon for page heading (appears to the left of the page heading's title) */
+  iconBeforeTitle?: React.ReactNode;
+  /** Optional icon for page heading (appears to the right of the page heading's title) */
+  iconAfterTitle?: React.ReactNode;
+  /** Optional label for page heading */
+  label?: {
+    name: string;
+    key?: string;
+    icon?: React.ReactNode;
+    href?: string;
+    color?: 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey';
+  };
+};
+
+export interface DetailsPageHeaderProps {
+  /** Top content area of details page */
+  pageHeading: PageHeading;
+   /** Navigational item that provides page context to help users navigate more efficiently and understand where they are in the application hierarchy */
+  breadcrumbs?: BreadcrumbProps[];
```

Comment:
This was required, but is now optional as I don't think we want to require breadcrumbs.